lib/routes/account.js

```diff
@@ -946,10 +1013,33 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+
+          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
+
+          if (doSigninConfirmation) {
+            return db.sessionTokenWithVerificationStatus(sessionToken.id)
```

Comment:
@rfk You mentioned looking up the verification status from the auth strategy, https://github.com/mozilla/fxa-auth-server/pull/1232#discussion_r63462831, however, IIUC that strategy will reject the request as [unauthorized](https://github.com/mozilla/fxa-auth-server/blob/48a6fa6335a29dab371d68c31507408ebd655a0d/lib/server.js#L69). Is there another way to get the verification status from auth strategy?

cc @philbooth